cli-commands/aeproject-epoch/epoch.js

```diff
@@ -65,6 +66,15 @@ async function fundWallets() {
   }
 }
 
+async function printBeneficiaryKey(client) {
```

Comment:
You can upgrade this function to print all of the wallets, with given parameters. That way we won't have repaeting code in this function and on row 63.